com.woltlab.wcf/templates/shared_interactionButton.tpl

```diff
@@ -1,9 +1,16 @@
-<div class="dropdown">
-	<button type="button" class="button small dropdownToggle" aria-label="{lang}wcf.global.button.more{/lang}">
-		{icon name='ellipsis'}
+<div class="dropdown {$configuration->cssClassName}">
+	<button
+		type="button"
+		class="dropdownToggle {$configuration->buttonCssClassName}{if !$configuration->label} jsTooltip{/if}"
+		{if !$configuration->label}title="{lang}{$configuration->tooltip}{/lang}"{/if}
+	>
+		{icon name=$configuration->icon size=$configuration->iconSize}
```

Comment:
This causes an unintentional limitations because this only supports non-solid icons and excludes the ability to use brand icons. Please see my remarks on the configuration class.